src/js/storyTelling-story.js

```diff
@@ -59,9 +65,40 @@ https://raw.githubusercontent.com/waharnum/sjrk-storyTelling/master/LICENSE.txt
             "{story}.events.onStoryListenToRequested":
             {
                 func: "{that}.queueSpeech",
-                args: ["{that}.model.storyText"]
+                args: ["@expand:{story}.getFullStoryText()"]
             }
         }
     });
 
+    // Generates full story text, including the title, author name and story content.
+    // title: the title of the story
+    // author: the author's name/moniker
+    // content: the textual content of the story
+    // bylineTemplate: the template string for the byline,
+    //      the value "%storyAuthor" will be replaced by the author's name
+    sjrk.storyTelling.story.getFullStoryText = function (title, author, content, bylineTemplate) {
+        var byLine = "";
+
+        if (author) {
+            if (bylineTemplate) {
+                byLine = bylineTemplate.replace("%storyAuthor", author);
+            } else {
+                byLine = "by " + author; // defaults to English
+            }
+        }
+
+        return (title ? title + " " : "") + byLine + (title || byLine ? ". " : "") + content;
+    };
+
+    // messagesString: JSON string of the collection of messages for the current locale, there should be
+    //      a "message_storyAuthorText" key. If not, it will return undefined
+    sjrk.storyTelling.story.getBylineTemplate = function (messagesString) {
+        if (messagesString) {
+            var messages = JSON.parse(messagesString);
+            return messages.message_storyAuthorText;
```

Comment:
We should avoid hard coding the expected key, and instead store the default of `message_storyAuthorText` on the component.